### PR TITLE
License finding curation reasons

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -178,12 +178,16 @@ e.g.:
   - path: "src/**/*.cpp"
     start_lines: "3"
     line_count: 11
-    detected_license: "Apache-1.1"
+    detected_license: "GPL-2.0-only"
+    reason: CODE,
```
Comment (sschuberth) [inline docs/Configuration.md]: The trailing comma looks wrong here, or? And mustn't "CODE" be quoted?

Comment (sschuberth) [reply docs/Configuration.md]: Ping @fviernau to ensure this does not get overlooked despite the PR being merged meanwhile.

Comment (fviernau) [reply docs/Configuration.md]: Good finding, fixing it now in a follow up PR.

Comment (fviernau) [reply docs/Configuration.md]: Quoting code seems not to be necessary. Do you prefer to have the quoting still? ...I could add it to the fix-up PR : https://github.com/heremaps/oss-review-toolkit/pull/1857

Comment (sschuberth) [reply docs/Configuration.md]: I'd prefer quoting if that's the way the config would be serialized. Looking at other enums like *project linkage* it seems that enums *do* get quoted usually, so yes, then I'd prefer that.

Comment (fviernau) [reply docs/Configuration.md]: Thanks, makese sense.

**Pull request merged**